src/Infrastructure/OrchestratorService.cs

```diff
@@ -104,7 +93,21 @@ public async Task FetchMapAndSaveEmployeeData()
             }
         }
 
-        var blobUrlsToBeDeleted = await _employeesRepository.EnsureEmployeesWithEndDateBeforeTodayAreDeleted();
+        // Remove employees with end date that has been passed.
+        var blobUrlsToBeDeleted = (await _employeesRepository.EnsureEmployeesWithEndDateBeforeTodayAreDeleted()).ToList();
+
+        foreach (var bemanning in bemanningEntries.Where(employee => !IsActiveEmployee(employee)))
+        {
+            // Remove potential employees that shouldn't have been added.

```

Comment:
And employees with endDate 😉

Reply:
Technically those would have been removed in the previous step and tried again (unnecessarily) to be removed here. Could potentially be explicit on IsBeforeStartDate

Reply:
Oh yeah right, read over that line 😅